src/runtime.md

```diff
@@ -51,7 +51,28 @@ defaults to unwinding the stack but that can be [changed to abort the
 process][abort]. The standard library's panic behavior can be modified at
 runtime with the [set_hook] function.
 
+## The `global_allocator` attribute
+
+The *`global_allocator` attribute* is used on a static item implementing the
+[`GlobalAlloc`] trait to set the global allocator.
+
+## The `windows_subsystem` attribute
+
+The *`windows_subsystem` attribute* may be applied at the crate level to
+indicate that when this crate is linked for a Windows target it will configure
+the resulting binary's [subsystem] via the linker. It uses the
+[_MetaNameValueStr_] syntax specify the subsystem with a value of either
+`console` or `windows`. More subsystems may be allowed in the future, and this
```

Comment:
This last sentence talks about the future in the first part and can be removed. The second part is an inverse of part of the first sentence. It can be removed.